### ISSUE TITLE
Gradle 3.4-rc-1 logs '/usr/bin/vm_stat' messages every 5 seconds when run with -i info logging flag

### ISSUE THREAD

**Issue by craigatk** [gradle/gradle#1270]
After upgrading a build from Gradle 3.3 to  Gradle 3.4-rc1 and running builds with the -i flag for info logging, every five seconds these two lines get logged to the console

```
Starting process 'command '/usr/bin/vm_stat''. Working directory: / Command: /usr/bin/vm_stat 
Successfully started process 'command '/usr/bin/vm_stat''
```

After a little searching I think this recent change to run a memory check every five seconds may be what's getting logged: https://github.com/gradle/gradle/commit/b64e1d0#diff-939973daee1670c3037fa7ff665cbbbcR59

### Expected Behavior
I often run Gradle commands with the -i flag to get more details on what is happening in the build and would prefer to not have the repeated vm_stat log messages adding noise to the log.

### Current Behavior
Every five seconds those two lines are logged to the console at the info level.

### Context
I run most of my Gradle commands with the -i flag to get more details on the build's progress.

### Steps to Reproduce (for bugs)
Run a long-ish running build with the -i flag using Gradle 3.4-rc-1

To reproduce the issue I took this project, upgraded it to Gradle 3.4-rc-1, and added a Thread.sleep(30000) in the test, and ran ```./gradlew -i build``` https://github.com/gradle/gradle-build-scan-quickstart

### Your Environment
Gradle 3.4-rc-1 (daemon enabled)
Oracle OpenJDK Runtime Environment 1.8.0_111-b14
Mac OS X 10.12.1

Build scan: https://scans.gradle.com/s/z4hzt6ldd4yga

**Comment by ghale** [comment]
To properly scope this, this happens only on MacOS and only when `--info` is used.  

What is interesting here is that these vmstat checks were occurring in 3.3, but as part of the daemon expiration check (every 10s).  For some reason, though, we didn't see these log messages with 3.3, but we do see them in 3.4 now that we have moved this to being published from `MemoryManager`.  I don't have an explanation for that.  

In any event, the longer term solution would be to implement @eskatos' native solution instead of spawning vmstat over and over.  If we need a short term solution, we'd probably want some way to tell `DefaultExecHandle` not to log starting/started messages for this particular execution.

**Comment by cuijianzhi** [comment]
me too!!
Android Studio stuck on “Gradle: resolve dependancies 'app:_debugCompile'” or 'app:_debugApk'

**Comment by bmuschko** [comment]
@cuijianzhi Please open another issue on GitHub that describes the problem you are seeing with resolving dependencies. I don't think it's related to this issue here.

**Comment by bhagyas** [comment]
This happens even without the info flag.

**Comment by cuijianzhi** [comment]
@bmuschko Ok.TKS!

**Comment by eskatos** [comment]
Merged, will be part of 3.5

**Comment by craigatk** [comment]
Thanks @eskatos !
